pkgs/unified_analytics/lib/src/constants.dart

```diff
@@ -51,6 +51,10 @@ reporting=1
 # a number representing the version of the message that was
 # displayed.''';
 
+/// Link to contextual survey metadata file
+const String kContextualSurveyUrl =
+    'https://docs.flutter.dev/f/contextual-survey-metadata.json';
```

Comment:
if we hadn't already published this, I would say do:

```json
{
  "surveys": [
    // as before...  
  ]
}
```

So that we're following the spec. However, I'm assuming `unified_analytics` is the only tool ever expected to parse this file? If so, and `dart:convert` is happy, then this is fine.